src/BaGet.Core/Configuration/BaGetOptions.cs

```diff
@@ -16,6 +16,10 @@ public class BaGetOptions
         /// </summary>
         public string PathBase { get; set; }
 
+        /// <summary>
+        /// The application Host without port number. If not set all hosts will be allowed. See https://github.com/aspnet/Announcements/issues/295
+        /// </summary>
+        public string HostUrl { get; set; }
```

Comment:
Let's align this with the `ForwardedHeadersOptions.AllowedHosts` property by renaming this to `AllowedHosts` and making it a list of strings.

Reply:
The way how BaGet configures its forwarded headers middleware has changed. I want to keep this pull request simple so if someone needs this feature, please open an issue.